os.go

```diff
@@ -193,3 +195,22 @@ func WriteFileSync(filename string, data []byte, perm uint32) []byte {
 	check(err)
 	return out
 }
+
+func StatSync(filename string) []byte {
+	info, err := os.Stat(filename)
```

Comment:
Use `afero.Stat`

Reply:
@qti3e Sure, no problem.

Thanks for reviewing.